packages/e2e-tests/specs/experiments/multi-entity-saving.test.js

```diff
@@ -177,11 +235,20 @@ describe( 'Multi-entity save flow', () => {
 			expect( enabledButton ).not.toBeNull();
 		} );
 
-		it( 'Clicking button should open modal with boxes checked', async () => {
+		it( 'save a11y button should be present', async () => {
+			await assertExistance( saveA11ySelector, true );
+		} );
+
+		it( 'Clicking button should open panel with boxes checked', async () => {
 			await page.click( activeSaveSiteSelector );
+			await page.waitForSelector( savePanelSelector );
 			await assertAllBoxesChecked();
 		} );
 
+		it( 'save a11y button should not be present with save panel open', async () => {
+			await assertExistance( saveA11ySelector, false );
+		} );
```

Comment:
I think we should test all 4 selectors here. Something like `assertCorrectButtonOrPanelShowing` could help.

Reply:
This section is in the context of the site editor, where the publish panel / a11y button are currently non-existent.  I don't think it would make sense at this point in time, and if we were to implement the publish panel here similarly at some point in the future it will have different selectors.  I think it would make sense to update that when the time comes?

Reply:
But it's the same component, right? I meant to guard against it showing it when it shouldn't.

Reply:
> But it's the same component, right? 

Not really.  For edit-post we have the <ActionsPanel /> component that returns the 4.

In edit-site we aren't using that.  We just have a ternary statement controlling whether we show the 1 panel or the 1 button.  The publish panel / button aren't imported into anything in use.